test/TestAsset/FlashMessages.php

```diff
@@ -42,6 +42,10 @@ public function getFlash(string $key, $default = null)
     {
     }
 
+    public function getFlashes() : array
+    {
+    }
```

Comment:
This method definition is wrong, must return `array`, but it's `void`. Not sure where (if) it is used?